frontend/composables/notifications/useNotificationsManagementDashboards.ts

```diff
@@ -137,7 +137,7 @@ export function useNotificationsManagementDashboards() {
         dashboard_id,
         group_id,
       },
-    )
+    ).then(() => refreshOverview())
```

Comment:
you are still breaking the functionality of `deleteDashboardNotifications` for `validator dashboards` as the `fetch` now returns the value of `refreshOverview()`

Reply:
:face_with_open_eyes_and_hand_over_mouth: Thank you for pointing this out, gonna take care of it `asap` :pray: